Content.Server/Temperature/Components/TemperatureComponent.cs

```diff
@@ -22,11 +22,26 @@ public class TemperatureComponent : Component
         /// <inheritdoc />
         public override string Name => "Temperature";
 
-        [ViewVariables] public float CurrentTemperature { get => _currentTemperature; set => _currentTemperature = value; }
+        [DataField("coldDamageType",required: true)]
+        private readonly string coldDamageType = default!;
+        [DataField("hotDamageType",required: true)]
+        private readonly string hotDamageType = default!;
+        [DataField("heatDamageThreshold")]
+        private float _heatDamageThreshold = default;
+        [DataField("coldDamageThreshold")]
+        private float _coldDamageThreshold = default;
+        [DataField("tempDamageCoefficient")]
+        private float _tempDamageCoefficient = 1;
+        [DataField("currentTemperature")]
+        private float _currentTemperature = Atmospherics.T20C;
+        [DataField("specificHeat")]
+        private float _specificHeat = Atmospherics.MinimumHeatCapacity;
 
+        [ViewVariables] public float CurrentTemperature { get => _currentTemperature; set => _currentTemperature = value; }
```

Comment:
The _currentTemperature field is also redundant here and can be removed and the DataField attribute moved to the new auto-property.